index.html

```diff
@@ -3045,7 +3069,16 @@ <h3>Trust Model</h3>
 [=Issuer=] does not need to know or trust the [=verifier=].
           </li>
         </ul>
-
+       <p>
+How [=verifiers=] decide which [=issuers=] to trust, and for what data or
+purposes, is out of scope for this recommendation. Some [=issuers=] (such
+as well-known organizations), might be trusted by many [=verifiers=] simply
+because of their reputation. Some [=issuers=] and [=verifiers=] might be
+members of a community in which all members trust each other due to the
+rules of membership. Some [=verifiers=] might trust a specific trust-service
+provider whose responsibility is to vet [=issuers=] and list them in a trust
+list such as those specified in [[[ETSI]]] [[ETSI]].
```

Comment:
Are there any other "trust lists" we could mention, so it doesn't appear that we're blessing ETSI in particular? Google suggests there might be both such lists, and lists of such lists, to [choose from](https://www.google.com/search?q=%22trust+list%22)...

Reply:
good idea. I can add Adobe trust list as well (https://helpx.adobe.com/uk/acrobat/kb/approved-trust-list1.html)

Reply:
If adding the **Adobe Approved Trust List Members**, I think the generic global URI (https://helpx.adobe.com/acrobat/kb/approved-trust-list1.html) would be better than any nationalized URI such as the UK link, above.